### PR TITLE
Feature/present only first and last name on allocation chart

### LINKED ISSUE #173: Present only first and last name on allocation chart
We should present only user's first and last name on the allocation chart page.

### PR DESCRIPTION
**Pull request by vinas-go**
Resolve #173

### CODE REVIEW

File: app/admin/contributions.rb [hunk 1]
```diff
@@ -88,7 +88,7 @@
       if current_user.super_admin?
         f.input :company
       else
-        f.input :company_id, as: :hidden, input_html: { value: current_user.company_id }
+        f.input :company_id, as: :hidden, input_html: {value: current_user.company_id}
```
Comment (wenderjean) [inline app/admin/contributions.rb:91]: Can we revert this change? :nail_care:

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -371,5 +371,41 @@
         end
       end
     end
+
+    describe '#first_and_last_name' do
```
Comment (wenderjean) [inline spec/models/user_spec.rb:375]: This change is on master already, I don't think it should be present in this PR :thinking:

**Comment by adbatista** [comment]
@vinas-go what is the status of this

**Comment by adbatista** [comment]
closing this PR seems to be abandoned, I'll address it to someone else
